examples/userdemo/colormap_interactive_adjustment.py

```diff
@@ -0,0 +1,52 @@
+"""
+========================================
+Interactive Adjustment of Colormap Range
+========================================
+
+Demonstration of using colorbar and picker functionality to make an
+interactively adjustable colorbar widget.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backend_bases import MouseButton
+
+
+def pick_fn(event):
```

Comment:
```suggestion
def on_pick(event):
```

is sort of a convention for event handling functions.